test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -158,7 +157,7 @@
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\requirements_PSGalleryOffline.psd1" />
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\requirements_PSGalleryOnline.psd1" />
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\profile.ps1">
-    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```

Comment:
nit: formatting issue here.

Reply:
yeah, a previous formatting issue that is now fixed! (it's meant to be indented) :-)